internal/net/ssl/ssl.go

```diff
@@ -162,13 +163,19 @@ func AddOrUpdateCertAndKey(name string, cert, key, ca []byte,
 		caFile.Write([]byte("\n"))
 		defer caFile.Close()
 
+		pemCertKey, err := fs.ReadFile(pemFileName)
```

Comment:
at https://github.com/Shopify/ingress/pull/68/files#diff-a3867fb2b922c407f56282c077f2177dR102 you avoid calling this function when dynamic cert mode enabled, why do you need this change here? Can`PemCertKey` not be empty when dynamic cert mode is not on?